packages/vow/src/E.js

```diff
@@ -311,7 +309,11 @@ export default makeE;
 /**
  * @template {Callable} T
  * @typedef {(
- *   (...args: Parameters<T>) => Promise<Unwrap<ReturnType<T>>>
+ *    ReturnType<T> extends EUnwrap<ReturnType<T>>
+ *    ? (...args: Parameters<T>) => Promise<ReturnType<T>>
+ *    : ReturnType<T> extends Promise<EUnwrap<ReturnType<T>>>
+ *    ? T
+ *    : (...args: Parameters<T>) => Promise<EUnwrap<ReturnType<T>>>
```

Comment:
without nesting this is hard to read. here's how prettier-plugin-jsdoc formats it:
```suggestion
 * @typedef {ReturnType<T> extends EUnwrap<ReturnType<T>>
 *     ? (...args: Parameters<T>) => Promise<ReturnType<T>>
 *     : ReturnType<T> extends Promise<EUnwrap<ReturnType<T>>>
 *       ? T
 *       : (...args: Parameters<T>) => Promise<EUnwrap<ReturnType<T>>>} ECallable
```

Reply:
Done.